config/settings.yml

```diff
@@ -89,6 +89,31 @@
     - Metric
     - MiqQueue
     - MiqWorker
+    :vacuum_schedule: "36 * * * *"
```

Comment:
What does this come out to in English? I thought it was weekly. Is this every hour on the 36th minute?

Reply:
@carbonin TBH I put this here and then forgot to update it. What should we put for periodic? Every other day? Every 2 days?

Reply:
I was thinking weekly. Maybe queue it Saturday early AM in case it runs long?